testworld/park.go

```diff
@@ -37,6 +37,7 @@ var hostConfig = []struct {
 	{"Charlie", 8086, 38206, true},
 	{"Kenny", 8087, 38207, false},
 	{"Eve", 8088, 38208, false},
+	{"Oskar", 8089, 38209, false},
```

Comment:
Because of your nice implementation, I see that we don't need `Oskar` at all. Since every new host consumes a lot of resources, we can remove Oskar and just use Eve for your tests right?